README.md

```diff
@@ -140,7 +160,7 @@ would look like for such an application.
 
 Your description:
 ```
-Replace this with your answer
+Restful routes are requests made to the server. They are composed of a method and an action.
```

Comment:
:+1:
I would have liked to have seen a little more detail here.  RESTful routes provide a convention for managing resources on the web.  We can expect a specific combination of http methods and paths.